media-gfx/png2ico/png2ico-2002.12.08.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="Converts PNG files to Windows icon resource files"
+DESCRIPTION="png2ico converts .PNG files to Windows .ICO icon resources"
+HOMEPAGE="http://winterdrache.de/freeware/png2ico/"
+COPYRIGHT="Free Software Foundation"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://winterdrache.de/freeware/png2ico/data/png2ico-src-${portVersion//./-}.tar.gz"
+CHECKSUM_SHA256="d6bc2b8f9dacfb8010e5f5654aaba56476df18d88e344ea1a32523bb5843b68e"
+SOURCE_DIR="png2ico"
+SOURCE_FILENAME_2="png2ico-$portVersion-memset_and_strcmp.patch"
+srcGitRev_2="8fb6e61574b0727120e3c7dcf4f8ed5b6f56d06c"
+SOURCE_URI_2="https://gitweb.gentoo.org/repo/gentoo.git/plain/media-gfx/png2ico/files/$SOURCE_FILENAME_2?id=$srcGitRev_2#noarchive"
+CHECKSUM_SHA256_2="643170b93549022d357ab5892f7f702f4ad0c0ad001ffbb784a1ad17d26a2390"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+PROVIDES="
+      png2ico= $portVersion
+      cmd:png2ico = $portVersion
+      "
+REQUIRES="
+      haiku
+      lib:libpng16
+      lib:libz
+      "
+
+BUILD_REQUIRES="
+      haiku_devel
+      devel:libpng16
+      devel:libz
+      "
+BUILD_PREREQUIRES="
+      cmd:gcc
+      cmd:make
+      cmd:install
```

Comment:
Oops, I forgot to tell that you can (and should) remove the **`cmd:install`** because it is provided by the coreutils package which is always available during a build.